ntbs-service-unit-tests/Services/TreatmentOutcomeServiceTest.cs

```diff
@@ -0,0 +1,488 @@
+﻿using System;
+using System.Collections.Generic;
+using ntbs_service.Models.Entities;
+using ntbs_service.Models.Enums;
+using ntbs_service.Models.ReferenceEntities;
+using ntbs_service.Services;
+using Xunit;
+
+namespace ntbs_service_unit_tests.Services
+{
+    public class TreatmentOutcomeServiceTest
+    {
+        private readonly ITreatmentOutcomeService _treatmentOutcomeService;
+
+        public TreatmentOutcomeServiceTest()
+        {
+            _treatmentOutcomeService = new TreatmentOutcomeService();
+        }
+
+        public static IEnumerable<object[]> NotificationsAndExpectedOutcome()
+        {
+            // Example 1
```

Comment:
Awkwardly going to suggest that the Example give an indication of the testing scenario. Not sure if this would be in cross-reference to some documentation somewhere (but where?) or a text description of the scenario.

Reply:
Maybe also a duplicated comment around expected outcome 12/24/36 to indicate that that is what they are